component/azstorage/config.go

```diff
@@ -406,6 +419,8 @@ func ParseAndValidateConfig(az *AzStorage, opt AzStorageOptions) error {
 		az.stConfig.authConfig.ClientID = opt.ClientID
 		az.stConfig.authConfig.ClientSecret = opt.ClientSecret
 		az.stConfig.authConfig.TenantID = opt.TenantID
+		az.stConfig.authConfig.ObjectID = opt.ObjectID
```

Comment:
This is still under SPN, we need to move it to MSI section or it shall be here ?